Makefile

```diff
@@ -29,7 +29,7 @@ DOCKER_IMAGE_REPO ?= openhorizon
 ANAX_K8S_OVERRIDE ?= ""
 
 ANAX_CONTAINER_DIR := anax-in-container
-DOCKER_IMAGE_VERSION ?= 2.23.10$(BRANCH_NAME)
+DOCKER_IMAGE_VERSION ?= 2.25.3$(BRANCH_NAME)
```

Comment:
Now that we no longer install the agent via apt, I think the version of the container needs to be controlled by the deb packager project, so this version should be deleted and you need to add support for the deb packager to produce the container, similar to what is done for the CSS and agent k8s containers.

Reply:
Updated the Makefile with @cgiroua. Also update Makefile and .travis.yaml for horizon-deb-packager https://github.com/open-horizon/horizon-deb-packager/pull/310.